### ISSUE TITLE
Sequence Generator -- System Reactive mismatch

### ISSUE THREAD

**Issue by brond3** [eXpandFramework/eXpand#708]
I added the SequenceGenerator via nuget.   Version 2.201.29.4 
It loaded all of the prerequisite with no Issues.  i.e. System.Reactive 4.4.1

I added this.RequiredModuleTypes.Add(typeof(Xpand.XAF.Modules.SequenceGenerator.SequenceGeneratorModule));  to the agnostic module. 

If I now try to view the module I get the error below indicating assembly System.Reactive Version 4.3.0.0 is missing. 

Any ideas why I am getting this error. 

----------StackTrace----------
   at Xpand.XAF.Modules.Reactive.ReactiveModuleBase..ctor()
   at Xpand.XAF.Modules.SequenceGenerator.SequenceGeneratorModule..ctor() in D:\a\1\s\src\Modules\SequenceGenerator\Module.cs:line 20
   at Create_SequenceGeneratorModule()
   at DevExpress.ExpressApp.Utils.Reflection.TypeData.CreateInstance(Object[] args)
----------InnerException----------
Could not load file or assembly 'System.Reactive, Version=4.3.0.0, Culture=neutral, PublicKeyToken=94bc3704cddfc263' or one of its dependencies. The system cannot find the file specified.
----------StackTrace----------
   at Xpand.XAF.Modules.Reactive.ReactiveModuleBase..cctor()

**Comment by apobekiaris** [comment]
> System.Reactive Version 4.3.0.0
it is not clear how u got this version the latest published lab version that u seem u used is on 4.3.2

```ps1
Get-NugetPackageDependencies Xpand.XAF.Modules.SequenceGenerator -Source Xpand

Id                         Include Exclude VersionRange
--                         ------- ------- ------------
Fasterflect.Xpand          {}      {}      [2.0.7, )
JetBrains.Annotations      {}      {}      [2019.1.3, )
System.Reactive            {}      {}      [4.3.2, )
Xpand.Extensions           {}      {}      [2.201.29.3, )
Xpand.Extensions.Reactive  {}      {}      [2.201.29.5, )
Xpand.Extensions.XAF       {}      {}      [2.201.29.5, )
Xpand.Extensions.XAF.Xpo   {}      {}      [2.201.29.1, )
Xpand.VersionConverter     {}      {}      [2.201.7, )
Xpand.XAF.Modules.Reactive {}      {}      [2.201.29.7, )
```

**Comment by brond3** [comment]
Could not get it working even after the System.Reactive was changed to 4.3.2.   In the end I started a brand new Solution.  This took care of the issue.  Thank you for your help.
